packages/deploy-tooling/src/base/validate.ts

```diff
@@ -237,7 +237,7 @@ async function getSystemPrefix(
         const atoSettings = await adtService.getAtoInfo();
         return atoSettings?.developmentPrefix;
     } catch (e) {
-        logger.error(e);
+        logger.error(e.message);
```

Comment:
Wondering if this should be switched to debug so that the full trace is shown if we need it? 

https://github.com/SAP/open-ux-tools/tree/main/packages/deploy-tooling#configuration-with-logging-enabled

So if verbose or higher is configured, we can see it?

Reply:
Based on our conversation, I would append the following too;
```
logger.debug(error);
```
So when the debug log level is set, we get more information if needed.

Reply:
Thanks @longieirl.